Make navtitle element from topictitle

PieterjanVandenweghe opened the pull request.

PieterjanVandenweghe committed:
Make navtitle element from topictitle

PieterjanVandenweghe committed:
Make navtitle element from topictitle

PieterjanVandenweghe committed:
Merge branch 'master' of https://github.com/PieterjanVandenweghe/dita-refactoring-examples.git

raducoravu commented:
@PieterjanVandenweghe thanks for wanting to make a contribution. Each pull request made to our repositories needs to have a signoff in the commit message.
Also looking in the stylesheet "SetnavtitleFromTopicTitle.xsl":

  - Maybe its name should be "SetNavtitleFromTopicTitle.xsl" in order to have consistent camel casing
  - I do not understand why this "[@audience]" selector appears, if the XML refactoring is to work generically, for any DITA project it should not contain a project specific case.
  - There is an xsl:message which should be removed: 
       &lt;xsl:message>test</xsl:message>
  - Does this XPath actually return the DITA topic title because it seems to return the text nodes from the topic root element?
      &lt;xsl:value-of select="document(@href)/topic[@audience]"/>

PieterjanVandenweghe commented:
Hi Radu,
Thank you for the feedback. 
It seems I uploaded not my latest version. 
I am sorry for that, but I'll take all of the comments into account before I upload again. 
Just wanted to combine two of your Refactoring scripts.

PieterjanVandenweghe committed:
- Replace @navtitle with <navitle> element based on the topic title for topicref and chapter elements.
- Update existing <navtitle> element based on the current topic title for topicref and chapter elements.

PieterjanVandenweghe commented:
@raducoravu  If you commit via the Client Add-onn from OxygenXML, does it also automatically sign off with my default user info? Or do I have to do it manually in the commit message?

raducoravu commented:
@PieterjanVandenweghe  our Git add-on does not have any signoff specific functionality so it needs to be manually added in the commit message.
